### PR TITLE
Add network observability CLI configs

### PR DESCRIPTION
**Pull request by jpinsonneau**
Adding https://github.com/netobserv/network-observability-cli to prow

### CODE REVIEW

**Comment by jpinsonneau** [comment]
@memodi @jotak can you please have a look ?

File: ci-operator/config/netobserv/network-observability-cli/netobserv-network-observability-cli-main.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+releases:
+  initial:
+    integration:
+      name: "4.16"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.16"
+      namespace: ocp
```
Comment (memodi) [inline ci-operator/config/netobserv/network-observability-cli/netobserv-network-observability-cli-main.yaml:28]: I figured we don't need multiple releases here, just one single nightly release should be sufficient, see my PR: https://github.com/openshift/release/pull/50006/files

Comment (jpinsonneau) [reply ci-operator/config/netobserv/network-observability-cli/netobserv-network-observability-cli-main.yaml:28]: Thanks for the input, I'll update this in my PR too :+1:

Comment (jpinsonneau) [reply ci-operator/config/netobserv/network-observability-cli/netobserv-network-observability-cli-main.yaml]: https://github.com/openshift/release/pull/50198/commits/f0c22b076d70abf8378bc079896950bc75313b08

**Comment by jpinsonneau** [comment]
Seems like Kalman is not in openshift org; removed him for now:
https://github.com/openshift/release/pull/50198/commits/cbe8c15889144fe6ec4afe33d186b313db587789

**Comment by jpinsonneau** [comment]
@danilo-gemoli & @jmguzik we are good to go here. Thanks !

**Comment by memodi** [comment]
@jpinsonneau - could you run rehearsals listed here https://github.com/openshift/release/pull/50198#issuecomment-2019989567 ?

**Comment by jpinsonneau** [comment]
/pj-rehearse pull-ci-netobserv-network-observability-cli-main-images

**Pull request merged**